### DISCUSSION TITLE
blog/permission-based-authorization-in-aspnet-core/

### DISCUSSION THREAD

**Discussion by giscus** [codewithmukesh/discussions#34] [General]
# blog/permission-based-authorization-in-aspnet-core/

Setting up Permissions to access your resources is always a crucial part of your application's security. In this article, we will implement Permission-Based Authorization in ASP.NET Core that builds upon the concept of Claim-Based Authorization in ASP.NET Core. As usual, we will be building this application right from scratch to get some detailed knowledge about the whole scenario and how it would actually help you secure your projects.

https://codewithmukesh.com/blog/permission-based-authorization-in-aspnet-core/

**Comment by kmorales026** [comment]
Nice Tutorial friends!! it is amazing!!!, so, How can I do if I want to create user/role from Register Page without using Seed ?
